styles/lib/states.css

```diff
@@ -0,0 +1,38 @@
+[data-severity="error"] {
+    border-left: 0.375rem solid var(--red);
+    background-color: var(--red-s3);
+}
+
+[data-severity="error"]:before {
```

Comment:
there is a lot of repeated css applying the same things.

Reply:
Fixed during team challenge, I will rename the error icon to allow a convention to be used.